### DISCUSSION TITLE
SMTP Connection Timed Out

### DISCUSSION THREAD

**Discussion by samiulice** [xhit/go-simple-mail#99] [Q&A]
error connecting mail server: Mail Error: SMTP Connection timed out
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x1 addr=0x28 pc=0xf72ee3]

goroutine 18 [running]:
github.com/xhit/go-simple-mail/v2.(*Email).SendEnvelopeFrom(0xc000461e18?, {0xc000076c60?, 0xbc43?}, 0x0?)
        C:/Users/samiu/go/pkg/mod/github.com/xhit/go-simple-mail/v2@v2.16.0/email.go:762 +0xa3
github.com/xhit/go-simple-mail/v2.(*Email).Send(...)
        C:/Users/samiu/go/pkg/mod/github.com/xhit/go-simple-mail/v2@v2.16.0/email.go:737
main.sendMsg({{0x10709f6, 0x1a}, {0xc0000da040, 0x1b}, {0x106f17c, 0x18}, {0x106d639, 0x16}, {0x1064fe8, 0xa}})

When server.Connect() is executed, the above error is thrown. Please anyone give me the solution

**Comment by xhit** [comment]
In the readme you have the usage.

Can you please share a minimal example triggering the error? Remove sensible information
